package.json

```diff
@@ -78,4 +77,4 @@
     "url": "https://www.snowflake.com/"
   },
   "license": "Apache-2.0"
-}
+}
```

Comment:
This is because there was a blank line at the last line of this file. In my changes, I removed it because I thought it was unnecessary. I made a space again